src/Plugin/WebformElement/MineOrganisationsData.php

```diff
@@ -567,6 +567,16 @@ private function getSearchResult(array &$element, string $query): array {
    *   The user ids if any.
    */
   private function getSearchUserIds(string $query): array {
+
+    // Remove extra whitespace.
+    $query = implode(' ', array_filter(explode(' ', $query)));
+
+    // Append wildcard character '*' to query string,
+    // if it is not already present.
+    if (!str_ends_with($query, '*')) {
+      $query .= '*';
+    }
+
```

Comment:
You should only add a wildcard if the query does not already contain a wildcard (or more wildcards), i.e. `Mikkel* Hest` should not have a(nother) wildcard added.

Reply:
This would result in `Jeppe Kuhl* And` not yielding me as an result. 

Perhaps it is fair to assume that any users using wildcards will know that the above example would not work.